### PR TITLE
New parameters

### PR DESCRIPTION
**Pull request by ben-kaufman**

### CODE REVIEW

File: parameters/production-parameters.json [hunk 1]
```diff
@@ -0,0 +1,100 @@
+{
```
Comment (orenyodfat) [inline parameters/production-parameters.json]: rename this file to production-bounty-dao.json

File: parameters/production-parameters.json [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+    "UpgradeScheme": false
+  },
+  "CustomSchemes": [
+    {
+      "name": "ReputationFromToken",
+      "isUniversal": false,
+      "params": [
+        "0x30412F4c06343d65C06B5a0Bba6821cf37E605fa",
+        { "StandAloneContract": 0 },
+        "0x0000000000000000000000000000000000000001"
```
Comment (orenyodfat) [inline parameters/production-parameters.json]: what's this param ?

File: parameters/production-parameters.json [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+  "ContributionReward": [{}],
+  "SchemeRegistrar": [{}],
+  "StandAloneContracts": [
+    {
+      "name": "PolkaCurve",
+      "fromArc": true,
+      "arcVersion": "0.0.1-rc.39"
+    },
+    {
+      "name": "ContributionRewardExt",
```
Comment (orenyodfat) [inline parameters/production-parameters.json]: this is standalone contract or custom ?

File: parameters/production-parameters.json (hunk 2 above)
Comment (ben-kaufman) [reply parameters/production-parameters.json]: The agreement hash

Comment (orenyodfat) [reply parameters/production-parameters.json]: please make it 0x0000000000000000000000000000000000000000

File: parameters/production-parameters.json (hunk 3 above)
Comment (ben-kaufman) [reply parameters/production-parameters.json]: There is an issue that both ContributionRewardExt and Competition are interdependent for their initialisation. To solve this, I deploy it in the standalone contracts but initialise in the custom schemes

File: parameters/production-parameters.json (hunk 1 above)
Comment (ben-kaufman) [reply parameters/production-parameters.json]: done

File: parameters/production-parameters.json (hunk 3 above)
Comment (orenyodfat) [reply parameters/production-parameters.json]: ok

File: parameters/production-bounty-dao.json [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+    "UpgradeScheme": false
+  },
+  "CustomSchemes": [
+    {
+      "name": "ReputationFromToken",
+      "isUniversal": false,
+      "params": [
+        "0x30412F4c06343d65C06B5a0Bba6821cf37E605fa",
+        { "StandAloneContract": 0 },
+        "0x0000000000000000000000000000000000000001"
```
Comment (orenyodfat) [inline parameters/production-bounty-dao.json]: please make it 0x0000000000000000000000000000000000000000

Comment (ben-kaufman) [reply parameters/production-bounty-dao.json]: done

**Pull request merged**